app/javascript/app/components/emission-pathways-table/emission-pathways-table.js

```diff
@@ -0,0 +1,71 @@
+import { PureComponent, createElement } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { getLocationParamUpdated } from 'utils/navigation';
+import qs from 'query-string';
+import PropTypes from 'prop-types';
+import { filteredDataBySearch } from './emission-pathways-table-selectors';
+import Component from './emission-pathways-table-component';
+
+const mapStateToProps = (state, { model }) => {
+  const search = qs.parse(location.search);
+  const EspData = {
+    state,
```

Comment:
I am not sure about sending the whole state. The store is quite big. Doesnt seem to have a bad effect but maybe it is more 'focused if you do that dynamic selection here. Not sure. Thoughts?